packages/core/src/electron-browser/menu/electron-main-menu-factory.ts

```diff
@@ -136,6 +136,11 @@ export class ElectronMainMenuFactory {
                     continue;
                 }
 
+                // We should omit rendering context-menu items which are disabled.
+                if (!this.commandRegistry.isEnabled(commandId, ...args) && !!isContextMenu) {
```

Comment:
[minor]: Can you swap the conditions? No need to check `isEnabled` when `isContextMenu` is falsy.